src/css/properties/css_modules.zig

```diff
@@ -100,17 +109,28 @@ pub const Specifier = union(enum) {
     /// The referenced name is global.
     global,
     /// The referenced name comes from the specified file.
-    file: []const u8,
+    ///
+    /// Is an import record index
+    file: u32,
     /// The referenced name comes from a source index (used during bundling).
-    source_index: u32,
+    // source_index: u32,
```

Comment:
If you meant to keep this commented out, then please delete it so we don't confuse it in the future.